crates/data/README.md

```
@@ -170,6 +170,6 @@ struct MyActor {
 ```
 
 We have also extracted the key calculation mechanism to a
-[`StoreKeys`](https://github.com/gnosisguild/interfold/blob/main/crates/config/src/store_keys.rs)
+[`StoreKeys`](https://github.com/theinterfold/interfold/blob/main/crates/events/src/store_keys.rs)
 struct. This is used in various places when creating repsitory factories for example
-[here](https://github.com/gnosisguild/interfold/blob/main/crates/aggregator/src/repositories.rs)
+[here](https://github.com/theinterfold/interfold/blob/main/crates/aggregator/src/repo.rs)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)